README.rst

```diff
@@ -65,6 +65,7 @@ pydoctor 20.12.1 (unreleased)
 
 * Reject source directories outside the project base directory (if given), instead of crashing
 * Fixed bug where source directories containing symbolic links could appear to be outside of the project base directory, leading to a crash
+* Bring back source link on package pages
```

Comment:
Very minor comment.

When we have full sentences, I prefer to have a full stop at the end. It's also a hint that the sentence was not accidentally left unfinished.

Maybe move this into a separate NEWS.rst /  Changelog.rst / ReleaseNotes.rst file

Reply:
I thought it was useful to have the release notes immediately visible on the GitHub page. But I underestimated how many changes we'd be making and the release notes became quite bulky, so I guess moving it to a separate document would make sense. We could do both full stops and separate file for the next major release.